packages/@adobe/spectrum-css-temp/components/fieldlabel/index.css

```diff
@@ -71,6 +71,18 @@ governing permissions and limitations under the License.
       /* If the user overrides the width of the field, propagate to the inner component */
       width: 100%;
     }
+
+    .spectrum-FieldLabel--quiet {
+      padding-bottom: var(--spectrum-fieldlabel-quiet-padding-bottom);
+      padding-top: var(--spectrum-fieldlabel-quiet-padding-top);
+      line-height: var(--spectrum-fieldlabel-quiet-text-line-height);
+    }
+
+    .spectrum-Field-fieldInput--quiet {
+      padding-top: var(--spectrum-textfield-quiet-padding-top);
+      padding-bottom: var(--spectrum-textfield-quiet-padding-bottom);
+      line-height: var(--spectrum-textfield-quiet-text-line-height);
+    }
```

Comment:
Values mimic what I saw in the XD file, but feels a bit off still. Doesn't feel like it matches the 20px mentioned in the ticket...